superset/db_engine_specs/presto.py

```diff
@@ -38,26 +44,34 @@
 
 
 class PrestoEngineSpec(BaseEngineSpec):
+    import pyhive.sqlalchemy_presto
```

Comment:
I think this is essentially the same as importing in module scope, it'll fail if `pyhive` isn't installed